ledfx/devices/__init__.py

```diff
@@ -38,6 +42,14 @@ class Device(BaseRegistry):
     def CONFIG_SCHEMA():
         return vol.Schema(
             {
+                vol.Required(
+                    "name", description="Friendly name for the device"
+                ): str,
+                vol.Required(
```

Comment:
From memory we had this lower down in the device class because WLED device types don't require this, since we attain the pixel count via an API call.